tools/run-bidi-server.mjs

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {spawn, spawnSync} from 'child_process';
+import {mkdirSync, createWriteStream} from 'fs';
+import {join, basename, dirname} from 'path';
+
+import {hideBin} from 'yargs/helpers';
+import yargs from 'yargs';
+
+process.chdir(dirname(dirname(process.argv[1])));
+
+function log(message) {
+  // eslint-disable-next-line no-console
+  console.log(`(${basename(process.argv[1])}) ${message}`);
+}
+
+const argv = yargs(hideBin(process.argv))
+  .usage(
+    `[CHANNEL=<stable | beta | canary | dev>] [DEBUG=*] [DEBUG_COLORS=<yes | no>] [HEADLESS=<true | false>] [LOG_DIR=logs] [NODE_OPTIONS=--unhandled-rejections=strict] [PORT=8080] $0`
+  )
+  .option('headless', {
+    describe:
+      'The --headless flag takes precedence over the HEADLESS environment variable.',
+    type: 'boolean',
+    default: process.env.HEADLESS === 'true',
+  })
+  .parse();
+
+let CHANNEL = process.env.CHANNEL || 'local';
+let BROWSER_BIN = process.env.BROWSER_BIN;
+
+if (CHANNEL === 'local') {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  BROWSER_BIN = spawnSync('node', ['tools/install-browser.mjs', '--shell'])
+    .stdout.toString()
+    .trim();
+  // Need to pass valid CHANNEL to the command below
+  CHANNEL = 'canary';
+}
+
+const DEBUG = process.env.DEBUG || '*';
```

Comment:
This `DEBUG` statement is incorrect. In bash, note that it is `${DEBUG-*}` instead of `${DEBUG:-*}`. This means that we must allow DEBUG to be empty.

I believe the correct node change is to replace `||` with `??`.